src/unstract/sdk/file_storage/__init__.py

```diff
@@ -6,8 +6,8 @@
     "SharedTemporaryFileStorage",
 ]
 
-from unstract.sdk.file_storage.fs_impl import FileStorage
-from unstract.sdk.file_storage.fs_permanent import PermanentFileStorage
-from unstract.sdk.file_storage.fs_provider import FileStorageProvider
-from unstract.sdk.file_storage.fs_shared_temporary import SharedTemporaryFileStorage
 from unstract.sdk.file_storage.helper import FileStorageHelper
+from unstract.sdk.file_storage.impl import FileStorage
+from unstract.sdk.file_storage.permanent import PermanentFileStorage
+from unstract.sdk.file_storage.provider import FileStorageProvider
+from unstract.sdk.file_storage.shared_temporary import SharedTemporaryFileStorage
```

Comment:
NIT: Can be added to `__init__.py` and imported from there for brevity

Reply:
You meean in sdk's __init__. I usually like to understand the logical nesting. Depends upon the user....